source/code-snippets/usage-examples/find.ts

```diff
@@ -0,0 +1,39 @@
+import { MongoClient } from "mongodb";
+
+// Replace the uri string with your MongoDB deployment's connection string.
+const uri =
+  "mongodb+srv://<user>:<password>@<cluster-url>?writeConcern=majority";
+
+const client = new MongoClient(uri);
+
+interface Movies {
```

Comment:
Why plural? Doesn't this interface represent one Movie?

Reply:
Changed to `Movie`